x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -586,17 +663,30 @@ export class ManifestManager {
   private async listEndpointPolicies(page: number) {
     return this.packagePolicyService.list(this.savedObjectsClient, {
       page,
-      perPage: 20,
+      perPage: 100,
       kuery: 'ingest-package-policies.package.name:endpoint',
     });
   }
 
-  private async listEndpointPolicyIds(page: number) {
-    return this.packagePolicyService.listIds(this.savedObjectsClient, {
-      page,
-      perPage: 20,
-      kuery: 'ingest-package-policies.package.name:endpoint',
-    });
+  private async listEndpointPolicyIds() {
```

Comment:
Question: I see we have a method above that returns all of the policies and this one returns only the IDs. Are we making double calls? could we get teh IDs from the call that is made to `listEndpointPolicies()`?